### PR TITLE
Change to other areas of the leaders page mundo

### PR DESCRIPTION
**Pull request by Shinsina**
Ths replaces PR #210

### CODE REVIEW

**Comment by Shinsina** [comment]
[How to solve for Mundo Leaders entries in Spanish and English](https://trello.com/c/oKzezqPm/50-how-to-solve-for-mundo-leaders-entries-in-spanish-and-english)

**Comment by Shinsina** [comment]
Screen Shot 2021-05-21 at 9 22 53 AM
Screen Shot 2021-05-21 at 9 10 50 AM

File: packages/common/components/blocks/company-details.marko [hunk 1]
```diff
@@ -12,12 +14,13 @@ $ const fields = [
   { key: 'trainingInformation', label: 'Training', tag: 'p' },
   { key: 'servicesProvided', label: 'Other services', tag: 'p' },
 ];
+$ const lang = input.lang || 'en'
```
Comment (solocommand) [inline packages/common/components/blocks/company-details.marko:17]: Use double quotes in template files (bonus points for fixing the definitions above too)

**Review by solocommand — CHANGES_REQUESTED** [review]

File: packages/shared/components/page-wrappers/content/leader.marko [hunk 2]
```diff
@@ -5,6 +5,7 @@ import queryFragment from "../../../graphql/fragments/content-list";
 
 $ const { site, i18n } = out.global;
 $ const { id, type, content } = input;
+$ const lang = site.config.lang || "en"
```
Comment (solocommand) [inline packages/shared/components/page-wrappers/content/leader.marko:8]: Where possible you should avoid accessing deeply nested properties and use a utility to retrieve them. Preferably this should be written as one of the following:
```js
import { defaultValue } from "@parameter1/base-cms-marko-web/utils";
$ const lang = defaultValue(site, "config.lang", "en");
```
or
```js
import { getAsObject } from "@parameter1/base-cms-object-path";
$ const { lang = "en" } = getAsObject(site, "config");
```

**Review by solocommand — CHANGES_REQUESTED** [review]

**Pull request merged**